packages/p2p/src/components/my-profile/payment-methods/add-payment-method/add-payment-method-form.jsx

```diff
@@ -22,6 +26,35 @@ const AddPaymentMethodForm = ({ should_show_separated_footer = false }) => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    React.useEffect(() => {
+        if (mutation_status === 'success') {
+            if (isCurrentModal('BlockUserModal')) {
+                hideModal();
+            }
```

Comment:
just wanted to clarify, how can the block user modal be the current modal being shown if we're within the add payment method form?

Reply:
I just moved out the logic from store `my_profile_store.createPaymentMethod` to avoid breaking anything 😆

Reply:
ahhhh, could you help to check if this would affect the flow of the add-payment-method-form if we remove this check? cause its a bit weird to me :P

Reply:
Yeah, it doest not make sense actually. I will try to remove and check back the behaviour